src/app/content/highlights/components/SummaryPopup/HighlightListElement.tsx

```diff
@@ -82,16 +83,23 @@ const HighlightListElement = ({ highlight, locationFilterId, pageId }: Highlight
   const [isDeleting, setIsDeleting] = React.useState(false);
   const dispatch = useDispatch();
 
+  const trackEditNoteColor = useAnalyticsEvent('editNoteColor');
+  const trackEditAnnotation = useAnalyticsEvent('editAnnotation');
+  const trackDeleteHighlight = useAnalyticsEvent('deleteHighlight');
+
   const updateAnnotation = (
     annotation: string
   ) => {
+    const addedNote = (highlight.annotation === undefined) ? true : false;
```

Comment:
`highlight.annotation === undefined` will already return `true` or `false` so ternary is not required